docs/source/examples/tfidf_example.rst

```diff
@@ -0,0 +1,143 @@
+
+TFIDF + scikit-learn SVM
+=========================
+
+In this example, we will cover a once popular family of models -- support vector machines (SVMs) with TF-IDF representations. As a simple example, we will analyse binary classification on the Stanford sentiment treebank (SST) dataset.
+
+First, we will implement a minimalistic example without much additional preprocessing. Since we're using TFIDF representation of our dataset, it is smart to limit the size of the vocabulary as each word needs to be present in the instance TFIDF representation. Let's load the SST dataset and convert it into a single batch:
+
+.. code-block:: python
+
+  >>> from podium import Vocab, Field, LabelField
+  >>> from podium.datasets import SST
+  >>> vocab = Vocab(max_size=5000, specials=())
+  >>> text = Field(name='text', numericalizer=vocab, disable_batch_matrix=True)
+  >>> label = LabelField(name='label')
+  >>> fields = {'text': text, 'label': label}
+  >>> train, dev, test = SST.get_dataset_splits(fields=fields)
+  >>> train.finalize_fields()
+  >>> x, y = train.batch(add_padding=True)
+
+We have now loaded our dataset, finalized its Fields and obtained it as a batch of input and target data. What we need to do next is define the TF-IDF vectorization for each instance in the dataset. This is done by using our :class:`podium.vectorizers.TfIdfVectorizer`, which adapts the ``scikit-learn`` vectorizer to the Podium input data.
+
+.. code-block:: python
+
+  >>> from podium.vectorizers import TfIdfVectorizer
+  >>> tfidf_vectorizer = TFIdfVectorizer()
+  >>> tfidf_vectorizer.fit(train, field=train.field('text'))
```

Comment:
Idea: add the option to pass an `str` as `field`. The `tfidf_vectorizer` could then default to extracting the field from the dataset itself. So it could be written as 
```python
tfidf_vectorizer.fit(train, field='text')
```
which IMO looks nicer.

Reply:
True, will add this change in a subsequent PR.